src/renderer/pages/Staking/ui/Staking.tsx

```diff
@@ -37,6 +37,13 @@ import { NetworkInfo } from './NetworkInfo';
 // eslint-disable-next-line import-x/max-dependencies
 import { NominatorsList } from './NominatorsList';
 
+// Lazy-loaded components
+const LazyBondExtra = lazy(() => import('@/widgets/Staking').then((module) => ({ default: module.BondExtra })));
+
+const LazyBondExtraShards = lazy(() =>
+  import('@/widgets/Staking').then((module) => ({ default: module.BondExtraShards })),
+);
```

Comment:
```suggestion
// Lazy-loaded components
const LazyBondExtra = lazy(() => import('@/widgets/Staking').then(({ BondExtra }) => ({ default: BondExtra })));

const LazyBondExtraShards = lazy(() =>
  import('@/widgets/Staking').then(({ BondExtraShards }) => ({ default: BondExtraShards })),
);
```

with destructuring you give bundler a chance to import only necessary entity instead of the whole module

with current approach you always import the whole module